README.md

```diff
@@ -10,7 +10,7 @@
   </a>
   <br>
   <p>
-    <em>Credits: <a href="https://dribbble.com/shots/3831443-Tech-Interview-Handbook">Illustration</a> by <a href="https://dribbble.com/yangheng">@yangheng</a></em>
+    <em>Credits: <a href="https://dribbble.com/shots/3831443-Tech-Interview-Handbook">Illustration</a> by <a href="https://dribbble.com/humongousheng">@yangheng</a></em>
```

Comment:
Thanks! Could you help change both the URL and the text to `@leftaligned`? I asked him and that's what he wants.

Reply:
No worries, and okay!

Reply:
Changed both URL and text to @leftaligned: #177